tests/cunit/CMakeLists.txt

```diff
@@ -233,6 +235,10 @@ else ()
     EXECUTABLE ${CMAKE_CURRENT_BINARY_DIR}/test_darray
     NUMPROCS ${AT_LEAST_FOUR_TASKS}
     TIMEOUT ${DEFAULT_TEST_TIMEOUT})
+  add_mpi_test(test_darray_uneven
```

Comment:
Well here is the issue. You added this test to the cmake build and it is failing.

Reply:
It works locally - how do I see details of the travis run?